zulipterminal/core.py

```diff
@@ -145,6 +145,7 @@ def spinning_cursor() -> Any:
 
         spinner = spinning_cursor()
         sys.stdout.write("\033[92mWelcome to Zulip.\033[0m\n")
+        sys.stdout.write(f"Running on {PLATFORM}.\n")
```

Comment:
This location does work, but note that most of the rest of the startup code is otherwise in `run.py`, for which we have tests, and for which having a small test for this would be reasonable too.

This approach is also supported by the fact that it'd likely be useful to see the platform at the top of the loading process? It doesn't necessarily have to be in the 'loading with' section, as that's historically been used just for controllable configuration, rather than an auto-detected platform.

Reply:
Okay, makes sense. I'll rellocate it to run.py.